P/PRIMME/build_tarballs.jl

```diff
@@ -0,0 +1,65 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "PRIMME"
+version = v"3.2.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/primme/primme.git",
+              "8d0ca1812436665564e5fba82d2a485e96fd7627"),
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+if [[ ${nbits} == 64 ]]; then
+CFLAGS="-O -fPIC -DNDEBUG -DPRIMME_BLASINT_SIZE=64 -DPRIMME_BLAS_SUFFIX=_64"
+else
+CFLAGS="-O -fPIC -DNDEBUG -DPRIMME_BLASINT_SIZE=32"
+fi
+
+if [[ "${target}" == *mingw* && ${nbits} == 32 ]]; then
+  LDFLAGS="-L${libdir} -lopenblas"
+elif [[ "${target}" == *mingw* && ${nbits} == 64 ]]; then
+  LDFLAGS="-L${libdir} -lopenblas64_"
+else
+  LDFLAGS="-L${libdir} -lblastrampoline"
+fi
+
+cd primme
+if [[ "${target}" == *mingw* ]]; then
+  sed -i 's/Windows\.h/windows.h/' ./src/linalg/wtime.c
```

Comment:
I still recommend submitting a PR upstream to fix this once and for all.

Reply:
I inadvertently hid the comment where I mentioned that.  They've already processed it: https://github.com/primme/primme/pull/67